shared/provision/forgot-username/index.tsx

```diff
@@ -1,93 +1,142 @@
-// TODO remove Container
 import * as React from 'react'
 import * as Kb from '../../common-adapters'
 import * as Styles from '../../styles'
-import Container from '../../login/forms/container'
-import * as Constants from '../../constants/provision'
+import PhoneInput from '../../signup/phone-number/phone-input'
+import {SignupScreen, errorBanner} from '../../signup/common'
 
 type Props = {
   forgotUsernameResult: string
   onBack: () => void
-  onSubmit: (email: string) => void
+  onSubmit: (email: string, phone: string) => void
+  waiting: boolean
 }
 
-type State = {
-  email: string
-}
+const ForgotUsername = (props: Props) => {
+  const [emailSelected, _setEmailSelected] = React.useState(true)
+  const [phoneSelected, _setPhoneSelected] = React.useState(false)
```

Comment:
You don't really need two state variables here, do you?

delete `   const [phoneSelected, _setPhoneSelected] = React.useState(false) `, the lines that call `_setPhoneSelected`, and just set `phoneSelected` to `!emailSelected`.

Reply:
Good call!